src/main/java/witchinggadgets/common/items/ItemClusters.java

```diff
@@ -158,4 +161,15 @@ public void getSubItems(Item item, CreativeTabs tab, List<ItemStack> itemList) {
                     itemList.add(new ItemStack(item, 1, iOre));
         }
     }
+
+    @Override
+    @Optional.Method(modid = "Railcraft")
+    public boolean shouldBurn(ItemStack itemStack) {
+        if (itemStack != null) {
+            int dmg = itemStack.getItemDamage();
+            return (dmg < witchinggadgets.common.WGContent.GT_Cluster.length
+                    && witchinggadgets.common.WGContent.GT_Cluster[dmg] == Materials.Firestone.mDefaultLocalName);
```

Comment:
please add a comment explaining why identity based string equality check is safe